src/pages/workspace/categories/ImportCategoriesPage.tsx

```diff
@@ -0,0 +1,21 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import ImportSpreedsheet from '@components/ImportSpreadsheet';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type ImportCategoriesPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES_IMPORT>;
+
+function ImportCategoriesPage({route}: ImportCategoriesPageProps) {
+    const policyID = route.params.policyID;
+
+    return (
+        <ImportSpreedsheet
```

Comment:
NAB: Is `ImportSpreedsheet` necessary when this component is doing nothing? We could combine the logic to live here.

Reply:
`ImportSpreadsheet` component will be used in a few places later